src/Chapter1.hs

```diff
@@ -631,8 +643,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let npositive = if n < 0 then n * (-1) else n
+        first = mod npositive 10
+        second = mod (div npositive 10) 10
+    in (first + second)
```

Comment:
You don't need parenthesis here 🙂 
Also, first and second are the names of standard functions, so maybe you could name these vars more precisely to omit conflicts 👌🏼